components/scream/src/physics/p3/p3_ic_cases.cpp

```diff
@@ -96,6 +96,8 @@ FortranData::Ptr make_mixed () {
     d.dzq(0,k) = Constants<double>::RD*T(k)/(g*d.pres(0,k))*dpres;
   }
 
+  // TODO - need data for pdel, exner, ast
```

Comment:
@AaronDonahue , need more initial condition setup here.

Reply:
@AaronDonahue , this should probably be resolved before you merge.

Reply:
absolutely, I pushed a new commit just now, @jgfouca , can you check to make sure that is the correct syntax?

Reply:
There was a minor syntax problem (** is not support in C++, use std::pow instead). And I removed the TODO since it's done now.